### PR TITLE
Make Server Timezone Configurable

### PR DESCRIPTION
**Pull request by raffaelschmid**
## Introduction
We are in timezone Europe/Zurich which has an offset of +2h in summer as well as +1h in winter. Unfortunately the database runs in timezone Europe/Zurich as well. 

However the result from Connection#getDBServerTimeZone is not timezone Europe/Zurich but a fixed +2h offset. 

Theoretically the local time of 2024-03-05 10:22:27 should be converted to 2024-03-05 09:22:27 (in winter offset is +1h) but ist converted to 2024-03-05 08:22:27. 

Please let me know if I can help with any further assistance or I am doing something completely wrong. 
Many thanks in advance. 

## TLDR
This MR adds a parameter "SERVER LOCATION" that lets the user configure the Server Timezone explicitly.

### CODE REVIEW

**Comment by sijms** [comment]
thank you for your contributions
